charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.36.0
+version: 9.36.1
```

Comment:
Hi @thefirstofthe300
This is a new feature, please make this a minor version bump.

Reply:
Fixed.